### ISSUE TITLE
unknown shorthand flag: 'f' in -f

### ISSUE THREAD

**Issue by slater1** [security-union/yew-actix-template#20]
I get the error in the title when running `make up`. Ubuntu 22.04 LTS.

```
sudo apt install docker
...
~/repos/yew-actix-template$ make up
docker compose -f docker/docker-compose.yaml up
unknown shorthand flag: 'f' in -f
See 'docker --help'.
```

**Comment by darioalessandro** [comment]
Hey @slater1 I suspect that this is due to the docker version that ubuntu ships with. 

I am using: 

```
darioalessandro@MacBook-Pro-di-Dario ~ % docker --version
Docker version 20.10.17, build 100c701
```
I recommend installing docker following the instructions here: https://docs.docker.com/desktop/install/ubuntu/

**Comment by slater1** [comment]
Hm, I've got a newer version

```
$ docker --version
Docker version 20.10.21, build 20.10.21-0ubuntu1~22.04.2
```

I did `sudo apt remove docker docker-desktop docker-compose` and `sudo apt purge docker-desktop`, followed the instructions you linked, and now it works.

TLDR: Don't use apt. (What else is new?)

**Comment by darioalessandro** [comment]
Glad it worked!!!
